### ISSUE TITLE
Google Drive folder path

### ISSUE THREAD

**Issue by afatac** [floccusaddon/floccus#1021]
I am trying to save the bookmark file in a folder (My Folder) in Google Drive.

[https://i.imgur.com/JVTyHY5.png](url)

However, it ends up saving as a file with the name My Folder/My floccus bookmark sync.xbel in the root folder in Google Drive.

**Comment by marcelklehr** [comment]
Hi @afatac 

Yep, that's a current limitation in the implementation, sadly. In order to get the file into that folder, you can let floccus create it and then move it to the correct folder.

**Comment by mvevitsis** [comment]
> Hi @afatac 
> 
> Yep, that's a current limitation in the implementation, sadly. In order to get the file into that folder, you can let floccus create it and then move it to the correct folder.

I can't get it to work. 
I created a file bookmarks.xbel
I moved it to Floccus/bookmarks.xbel
Then I changed the path in Floccus to Floccus/bookmarks.xbel, but it creates a file in root again.

**Comment by marcelklehr** [comment]
Does it work if you keep the setting as `bookmarks.xbel` in floccus even though you moved it?

**Comment by mvevitsis** [comment]
> Does it work if you keep the setting as `bookmarks.xbel` in floccus even though you moved it?

Yes that works. I made it and moved it after and everything is fine. And I can import in other browsers and on mobile.

**Comment by Jondaman777** [comment]
I was able to get it to work if you use a ">". Instead of a forward slash. 

For example -  Floccus>bookmarks.xbel 

This will create the bookmarks.xbel file in the folder labeled Floccus in your Google Drive.
Or you can edit the path later if need be.

**Comment by eldorel** [comment]
> I was able to get it to work if you use a ">". Instead of a forward slash.
> 
> For example - Floccus>bookmarks.xbel
> 
> This will create the bookmarks.xbel file in the folder labeled Floccus in your Google Drive. Or you can edit the path later if need be.

just as a note: I just tried this on chrome and was not able to get it to work.
